src/routes/areas/deleteArea.ts

```diff
@@ -3,30 +3,34 @@ import {
   RawRequestDefaultExpression,
   RawServerDefault,
   RouteHandler,
-  RouteOptions
-} from 'fastify';
-import AreaSchema from '../../schemas/Area.json';
-import { AreaRequestParams } from '../../types/AreaRequestParams';
-import AreaRequestParamsSchema from '../../schemas/AreaRequestParams.json';
-import { Area } from '../../types/Area';
-import AreaModel from '../../models/area';
+  RouteOptions,
+} from "fastify";
+import AreaSchema from "../../schemas/Area.json";
+import { AreaRequestParams } from "../../types/AreaRequestParams";
+import AreaRequestParamsSchema from "../../schemas/AreaRequestParams.json";
+import { Area } from "../../types/Area";
+import AreaModel from "../../models/area";
+import { ErrorSchema } from "../../types/ErrorSchema";
+import ErrorSchemaJson from "../../schemas/ErrorSchema.json";
 
-type Reply = {};
+type Reply = ErrorSchema;
 type DeleteAreaRoute = {
   Body: Area;
   Params: AreaRequestParams;
   Reply: Reply;
 };
 
-const url = '/area/:id';
+const url = "/area/:id";
 
 const handler: RouteHandler<DeleteAreaRoute> = async (req, reply) => {
   const areaId = req.params.id;
```

Comment:
Destructuring id with an alias...
```suggestion
const {id:areaId} = req.params;
```

Reply:
it's a [nitpick](https://conventionalcomments.org/) @leandro0day (non-blocking by nature)